app/models/manageiq/providers/ansible_runner_workflow.rb

```diff
@@ -22,11 +22,25 @@ def execution_type
     raise NotImplementedError, "must be implemented in a subclass"
   end
 
+  def start
+    queue_signal(:pre_execute)
```

Comment:
@carbonin I think I have a way to remove the wait_for_task thing, by exposing queue_signal (or a variant of it) and allowing one to override the role.  Then the caller when it does `create_job { |j| j.signal(:start) }` can instead do `create_job { |j| j.queue_signal(:start, :role => "embedded_ansible") }`

Reply:
So in this case we would get rid of queueing entirely at the ansible playbook service level, right?

I like that because it's more similar to what we were doing before the tower => runner change where the actual tower job launch was async and the workflow was responsible for starting and monitoring it.

Reply:
Well the queueing would still exist, but we'd let the workflow take care of it, and it would avoid the double-queueing (create a queue item for something that starts by creating a queue item).  I'd love to even make queue_signal a generic Job method to support this use case.

Reply:
:+1: I like it.